src/Sylius/Bundle/ApiBundle/Resources/config/api_resources/ProductOptionTranslation.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" ?>
+
+<resources xmlns="https://api-platform.com/schema/metadata"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xsi:schemaLocation="https://api-platform.com/schema/metadata https://api-platform.com/schema/metadata/metadata-2.0.xsd">
+
+    <resource
+        class="Sylius\Component\Product\Model\ProductOptionTranslation"
+        shortName="ProductOptionTranslation"
+    >
+        <attribute name="validation_groups">sylius</attribute>
+        <collectionOperations />
+        <itemOperations>
+            <itemOperation name="get" />
+        </itemOperations>
+        <property name="id" identifier="true" writable="false" />
+        <property name="name" identifier="true" required="true" />
```

Comment:
Why we have 2 identifier? It seems that we can get resource by name or id ?

Reply:
It should not be an identifier for sure 🎉 🚀